### PR TITLE
feat: allow ignoring workspace members

### LINKED ISSUE #123: Allow to exclude workspace' packages by paths
[Now that typesync works seamlessly in monorepos](https://github.com/jeffijoe/typesync/pull/119), there are multiple package.json that gets scanned.
But it would be very useful to have the option to exclude certain projects/packages by path.

```ts
// .typesyncrc
{
  "ignoreProjects": ["apps/acme-docs"]
}

### PR DESCRIPTION
**Pull request by lishaduck**
Fixes #123, allow to exclude workspace' packages by paths.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ export interface ISyncOptions {
    * Ignore certain packages.
    */
   ignorePackages?: Array<string>
+
+  ignoreProjects?: IWorkspacesArray
```
Comment (jeffijoe) [inline src/types.ts:23]: Let's add a doc block here

**Review by jeffijoe — COMMENTED** [review]
Would you be able to add a test for this as well?

**Comment by lishaduck** [comment]
> Would you be able to add a test for this as well?

I modified the existing suite to cover it, and it currently hits 100%.
Do you just want a specific test?

**Comment by jeffijoe** [comment]
@lishaduck my bad I must have missed it!

**Review by jeffijoe — APPROVED** [review]

**Comment by jeffijoe** [comment]
Thanks @lishaduck ! I've released 0.13.1!

**Pull request merged**